packages/test-app/tests/integration/components/editable-field-test.js

```diff
@@ -152,9 +150,22 @@ module('Integration | Component | editable field', function (hooks) {
       hbs`<EditableField @close={{(noop)}} @value='lorem' @onEditingStatusChange={{set this 'status'}} />`,
     );
     assert.notOk(this.status);
-    await click('[data-test-edit]');
+    await component.editable.edit();
     assert.ok(this.status);
-    await click('[data-test-cancel]');
+    await component.cancel();
     assert.notOk(this.status);
   });
+
+  test('it has title property value only if @showTitle is true', async function (assert) {
+    this.intl = this.owner.lookup('service:intl');
```

Comment:
Nice, but not standard for our tests. If you use `setupIntl(hooks, 'en-us');` it will ensure the language is set to English and you can just test that value. Or if you want to get really fancy, use the [t test helper](https://ember-intl.github.io/ember-intl/docs/test-helpers/t)